src/test/java/com/synopsys/integration/detect/workflow/report/FormattedOutputManagerTest.java

```diff
@@ -1,5 +1,47 @@
-// TODO Jordan deleted this test. It would require a re-write
+package com.synopsys.integration.detect.workflow.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.detect.configuration.DetectInfo;
+import com.synopsys.integration.detect.configuration.enumeration.DetectTool;
+import com.synopsys.integration.detect.configuration.enumeration.ExitCodeType;
+import com.synopsys.integration.detect.workflow.event.Event;
+import com.synopsys.integration.detect.workflow.event.EventSystem;
+import com.synopsys.integration.detect.workflow.report.output.FormattedCodeLocationOutput;
+import com.synopsys.integration.detect.workflow.report.output.FormattedOutput;
+import com.synopsys.integration.detect.workflow.report.output.FormattedOutputManager;
+import com.synopsys.integration.detect.workflow.status.FormattedCodeLocation;
+
+public class FormattedOutputManagerTest {
+    @Test
+    public void formattedCodeLocationTest() {
+        // Setup the reporting infrastructure
+        EventSystem eventSystem = new EventSystem();
+        FormattedOutputManager formattedOutputManager = new FormattedOutputManager(eventSystem);
+        
+        // Mock a completed scan to report on
+        List<FormattedCodeLocation> codeLocationExpected = new ArrayList<>();
+        codeLocationExpected.add(new FormattedCodeLocation(null, "f4ffe78c-9c83-4f45-b66e-d60fd2fcce46", DetectTool.DETECTOR.name()));
```

Comment:
I think it would be a good idea to use `UUID.randomUUID` here instead of the static scan ID string in case there are any validations in the workflow in future that check if it's a valid UUID.